src/js/modules/helpers.js

```diff
@@ -0,0 +1,15 @@
+export function loadAllSliderImages(sectionClass) {
+
+	const observer = new IntersectionObserver(([entry], observer) => {
+
+		if (entry.isIntersecting) {
+			const section = document.querySelector(sectionClass);
+			const images = section.querySelectorAll("[loading='lazy']");
+
+			images.forEach((image)=> image.setAttribute("loading", "eager"));
+			observer.unobserve(entry.target);
+		}
+	});
+
+	observer.observe(document.querySelector(sectionClass));
```

Comment:
I see you use `document.querySelector(sectionClass)` 2 times in one function call. Create variable for it.

Reply:
Done